lib/tags.js

```diff
@@ -0,0 +1,14 @@
+module.exports.generateTagsSpec = tags => {
+  return tags.map(t => {
+    return {
+      name: t,
+      description: `${t} resource`
+    };
+  });
+};
+
+module.exports.matchingTags = (tagsSpec, path) => {
+  return tagsSpec.filter(t => {
+    return path.toLowerCase().includes(t.name.toLowerCase());
```

Comment:
Not sure if problems could arise due to this matching quite loosely.

I don't know much about how tags should work and what are the common/expected use cases, so use your best judgement:D

A problem I can think of is perhaps if there're separate resources like `User` and `Users` and the developer would want them separate but they'd get mashed together. Once again, not sure if this would ever be a problem, just contemplating c:

Reply:
Mostly tags are used to group by resources. I really cannot see a real use case in which you would have the singular and plural resources as different things. 
Even if that's the case and you provide both tags as `['User', 'Users']` the plural paths will be tagged by both.

Reply:
Great, makes sense, leggo 🚀